target/linux/ramips/dts/mt7620a_iptime_a104ns.dts

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a_iptime.dtsi"
+
+/ {
+	compatible = "iptime,a104ns", "ralink,mt7620a-soc";
+	model = "ipTIME A104ns";
+
+	aliases {
+		led-boot = &led_cpu;
+		led-failsafe = &led_cpu;
+		led-running = &led_cpu;
+		led-upgrade = &led_cpu;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		usb {
+			label = "a104ns:blue:usb";
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&ohci_port1>, <&ehci_port1>;
+			linux,default-trigger = "usbport";
+		};
+
+		led_cpu: cpu {
+			label = "a104ns:blue:cpu";
+			gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
Really? They just use the inverted gpio<->led assignment compared to the A1004ns?

Reply:
Yes, I can confirm this, and even vendor GPL source code proves it. How weird.
https://github.com/iptime-gpl/userapps_a104ns/blob/master/fs/mt7621/clones/a104ns/si/hw#L24
https://github.com/iptime-gpl/userapps_a1004ns/blob/master/fs/mt7621/clones/a1004ns/si/hw#L25